### PR TITLE
Introduce Arc / Lyra automation resources

### LINKED ISSUE #5: Add support for CCloud automation management
Hello, colleagues. 

This is a feature request to introduce support for automation management in CCloud with Terraform: it is desired to:

- Add instances to automation at the time of instance creation;
- Manage automations (create / modify / delete) 

Thanks!

### PR DESCRIPTION
**Pull request by kayrus**
Resolves #5

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,19 +4,19 @@ services:
 - docker
 language: go
 go:
-- 1.8.1
+  - "1.11.x"
```
Comment (BugRoger) [inline .travis.yml]: Do the other providers still build? 😛

File: .travis.yml [hunk 2]
```diff
@@ -4,19 +4,19 @@ services:
...
+env:
+  - GOFLAGS=-mod=vendor GO111MODULE=on
```
Comment (BugRoger) [inline .travis.yml]: Does the whole Limes dependency tree still work?

Comment (kayrus) [reply .travis.yml]: yes it does

File: .travis.yml (hunk 1 above)
Comment (kayrus) [reply .travis.yml]: I copied it from the latest https://github.com/terraform-providers/terraform-provider-openstack/blob/master/.travis.yml

**Comment by kayrus** [comment]
@BugRoger yes, the documentation is here: https://github.com/kayrus/terraform-provider-ccloud/tree/cc-automation/website/docs

**Comment by kayrus** [comment]
DuWw9ilWoAAxibX

**Pull request merged**